edumanage/context_processors.py

```diff
@@ -18,6 +18,7 @@ def country_code(context):
         'GOOGLE_MAPS_API_KEY': settings.GOOGLE_MAPS_API_KEY if hasattr(settings,"GOOGLE_MAPS_API_KEY") else None,
         'SHIB_AUTH_ENTITLEMENT': settings.SHIB_AUTH_ENTITLEMENT if hasattr(settings,"SHIB_AUTH_ENTITLEMENT") else None,
         'FEDERATION_DOC_URL': settings.FEDERATION_DOC_URL if hasattr(settings,"FEDERATION_DOC_URL") else None,
+        'OUR_TLSPSK_REALM': settings.NRO_TLSPSK_REALM if hasattr(settings,"NRO_TLSPSK_REALM") else 'localhost',
```

Comment:
How about putting the default value for `NRO_TLSPSK_REALM` into `djnro/settings.py`, with site-specific value being configured in `djnro/local_settings.py` ?

The settings in DjNRO have been designed specifically for this - that local_settings.py can override values from settings.py.

This would avoid the need to have the default value hard-coded in 3 different places - and the code would be able to use `settings.NRO_TLSPSK_REALM` without having to check if it is defined, as `setting.py` comes with the code.

Reply:
This one I'm in two minds about this. We wouldn't ever want anyone leaving this as `localhost` in production; that *should* be an error. So maybe removing the defaulting is better? Or maybe we need to make the default in settings.conf something obviously wrong in a reserved domain.

Reply:
I realise that a default must be set for people upgrading, so it's now deliberately invalid and uses an RFC2606 suffix.